<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>625</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>E/D - Fg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(195, 209, 220);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="browseButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(25, 25, 25);
color: rgb(241, 241, 241);</string>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="selectedFileLabel">
        <property name="minimumSize">
         <size>
          <width>475</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(25, 25, 25);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QPushButton" name="addToFilesButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(241, 241, 241);
color: rgb(25, 25, 25);</string>
        </property>
        <property name="text">
         <string>Add To Files</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="clearFilesListButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(241, 241, 241);
color: rgb(25, 25, 25);</string>
        </property>
        <property name="text">
         <string>Clear Files List</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="filesPathBox">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(241, 241, 241);
color: rgb(25, 25, 25);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QPushButton" name="savePathButton">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(25, 25, 25);
color: rgb(241, 241, 241);</string>
        </property>
        <property name="text">
         <string>Save Files Path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="savePathLabel">
        <property name="minimumSize">
         <size>
          <width>475</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(25, 25, 25);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLineEdit" name="passwordBox">
        <property name="minimumSize">
         <size>
          <width>435</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(241, 241, 241);
color: rgb(25, 25, 25);</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>134</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(25, 25, 25);</string>
        </property>
        <property name="text">
         <string>Enter Password:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="unmaskPasswordButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 8pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(241, 241, 241);
color: rgb(25, 25, 25);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="confirmPasswordLabel">
        <property name="minimumSize">
         <size>
          <width>134</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(25, 25, 25);</string>
        </property>
        <property name="text">
         <string>Confrim Password:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="confirmPasswordBox">
        <property name="minimumSize">
         <size>
          <width>435</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(241, 241, 241);
color: rgb(25, 25, 25);</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="unmaskConfirmPasswordButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 8pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(241, 241, 241);
color: rgb(25, 25, 25);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="encryptButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(250, 70, 20);
color: rgb(25, 25, 25);</string>
        </property>
        <property name="text">
         <string>Encrypt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="encryptAllButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(250, 70, 20);
color: rgb(25, 25, 25);</string>
        </property>
        <property name="text">
         <string>Encrypt All</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="decryptButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(0, 33, 165);
color: rgb(25, 25, 25);</string>
        </property>
        <property name="text">
         <string>Decrypt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="decryptAllButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe MDL2 Assets&quot;;
background-color: rgb(0, 33, 165);
color: rgb(25, 25, 25);</string>
        </property>
        <property name="text">
         <string>Decrypt All</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="errorLayout">
      <item>
       <widget class="QLabel" name="errorLabel">
        <property name="text">
         <string>Passwords do not match!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
